lightly_studio/src/lightly_studio/resolvers/embedding_model_resolver.py

```diff
@@ -34,8 +34,6 @@ def get_or_create(session: Session, embedding_model: EmbeddingModelCreate) -> Em
         db_model.name != embedding_model.name
         or db_model.parameter_count_in_mb != embedding_model.parameter_count_in_mb
         or db_model.embedding_dimension != embedding_model.embedding_dimension
-        # TODO(Michal, 09/2025): Allow same model for different datasets.
-        or db_model.dataset_id != embedding_model.dataset_id
```

Comment:
This is a plot twist: EmbeddingModel has dataset_id.

It is not clear what is the intended use. Currently it is legacy because we were removing dataset_ids from tables.

There are two possible interpretations:
* It is the "root dataset id" - with multiple (root) datasets the model belongs only to a single one.
* It is a "collection id" - the model belongs to a collection

I think the first interpretation makes more sense at the moment. WDYT? What do you think we should do with the field?